requirements.txt

```diff
@@ -1,15 +1,13 @@
-numpy == 1.26.3
-scipy == 1.11.4
+numpy == 1.26.4
+scipy == 1.13.1
 matplotlib == 3.7.*
-qiskit == 0.45.1
-qiskit-dynamics == 0.4.2
-qiskit-experiments == 0.5.4
-qiskit-experiments == 0.5.4
-qiskit-ibm-provider == 0.8.0
-qiskit-ibm-runtime == 0.17.0
-qiskit-terra == 0.45.1
+qiskit == 1.1.1
+qiskit-dynamics == 0.5.1
+qiskit-experiments == 0.7.0
+qiskit-ibm-runtime == 0.24.1
+qiskit-ibm-experiment == 0.4.7
 uncertainties
 lmfit
 pylatexenc
 qutip == 5.0.*
-jax[cpu] == 0.4.23
+jax[cpu] == 0.4.6
```

Comment:
Why is jax downgraded to `0.4.6`?

In fact, even `jax[cpu]==0.4.23` has a conflict with  `scipy>1.12` I think. I got the error.


```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Local/bli/qiskit/qiskit-CR-calibration/qiskit_utilities/__init__.py", line 5, in <module>
    from .fake_backend import *
  File "/Local/bli/qiskit/qiskit-CR-calibration/qiskit_utilities/fake_backend.py", line 10, in <module>
    from qiskit_dynamics.array import Array
  File "/local/bli/conda/qiskit1/lib/python3.11/site-packages/qiskit_dynamics/__init__.py", line 38, in <module>
    from .solvers.solver_functions import solve_ode, solve_lmde
  File "/local/bli/conda/qiskit1/lib/python3.11/site-packages/qiskit_dynamics/solvers/__init__.py", line 110, in <module>
    from .solver_functions import solve_ode, solve_lmde
  File "/local/bli/conda/qiskit1/lib/python3.11/site-packages/qiskit_dynamics/solvers/solver_functions.py", line 39, in <module>
    from .fixed_step_solvers import (
  File "/local/bli/conda/qiskit1/lib/python3.11/site-packages/qiskit_dynamics/solvers/fixed_step_solvers.py", line 34, in <module>
    from jax.scipy.linalg import expm as jexpm
  File "/local/bli/conda/qiskit1/lib/python3.11/site-packages/jax/scipy/linalg.py", line 18, in <module>
    from jax._src.scipy.linalg import (
  File "/local/bli/conda/qiskit1/lib/python3.11/site-packages/jax/_src/scipy/linalg.py", line 409, in <module>
    @_wraps(scipy.linalg.tril)
            ^^^^^^^^^^^^^^^^^
AttributeError: module 'scipy.linalg' has no attribute 'tril'
```